crates/mun_compiler/src/driver.rs

```diff
@@ -1,27 +1,31 @@
 //! `Driver` is a stateful compiler frontend that enables incremental compilation by retaining state
 //! from previous compilation.
 
-use crate::{
-    db::CompilerDatabase,
-    diagnostics::{diagnostics, Emit},
-    PathOrInline,
-};
+use crate::{db::CompilerDatabase, diagnostics::diagnostics, PathOrInline};
 use mun_codegen::{IrDatabase, ModuleBuilder};
 use mun_hir::{FileId, RelativePathBuf, SourceDatabase, SourceRoot, SourceRootId};
+
 use std::{path::PathBuf, sync::Arc};
 
+mod color;
 mod config;
 
+pub use self::color::Color;
 pub use self::config::Config;
-use mun_errors::{Diagnostic, Level};
-use termcolor::WriteColor;
+
+use annotate_snippets::{
+    display_list::DisplayList,
+    formatter::DisplayListFormatter,
+    snippet::{AnnotationType, Snippet},
+};
 
 pub const WORKSPACE: SourceRootId = SourceRootId(0);
 
 #[derive(Debug)]
 pub struct Driver {
     db: CompilerDatabase,
     out_dir: Option<PathBuf>,
+    color_option: Color,
```

Comment:
This makes me question the naming of the `Color` type. I'd expect it to be a colour, but it is actually an option